io/plugins/eu.esdihumboldt.hale.io.csv/plugin.xml

```diff
@@ -105,6 +105,17 @@
                   default="\">
             </valueDescriptor>
          </providerParameter>
+         <providerParameter
+               description="Divisor is used to identify float value"
```

Comment:
*Divisor* sounds strange to me in that context when talking about numbers. Wouldn't `separator` be [more appropriate](http://dict.leo.org/ende/index_de.html#/search=Dezimaltrenner&searchLoc=0&resultOrder=basic&multiwordShowSingle=on&pos=0)?